db/external_sst_file_basic_test.cc

```diff
@@ -351,6 +353,72 @@ TEST_F(ExternalSSTFileBasicTest, BasicWithFileChecksumCrc32c) {
   DestroyAndRecreateExternalSSTFilesDir();
 }
 
+TEST_F(ExternalSSTFileBasicTest, KeyValueOrdering) {
+  SyncPoint::GetInstance()->SetCallBack(
+      "SSTFileWriter::Add", [&](void* arg) {
+          Slice key = *static_cast<Slice*>(arg);
```

Comment:
Minor: it feels more straight forward to do `Slice&` rather than making a copy here, although it is correct.